### DISCUSSION TITLE
table list page, how search table

### DISCUSSION THREAD

**Discussion by ipfred** [open-metadata/OpenMetadata#16873] [Q&A]
How to search for a specific table on the table list,  turn the page to view it is not practical.
eg
![](https://raw.githubusercontent.com/ipfred/my_pics/main/picgo/2404/20240702100910.png)

**Comment by Prajwal214** [comment]
@ipfred You can raise a feature request for the same, In the mean time you can search the table from the search bar.

<img width="1188" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/167504578/fb203b6a-3f1c-4927-bc7a-89eb3fc24efa">
